src/data/finset/lattice.lean

```diff
@@ -991,6 +991,9 @@ begin
     refl }
 end
 
+lemma count_sup' [decidable_eq α] (x : α) (s t : multiset α) :
+  count x (s ⊔ t) = max (count x s) (count x t) := by simp
```

Comment:
I think this belongs in multiset/basic, as from what I can tell it doesn't mention finsets at all